### PR TITLE
#163342847 View user and admin signin page

### PR DESCRIPTION
**Pull request by captainPrime**
**What does the PR do?**

Shows the signin page for the CRUD application

**Description of Task to be completed?**

Here i have the page working:

`file:///C:/Users/dude/Desktop/bootcamp/UI/signin.html`

**How should it be manually tested?**

After cloning the repo, cd into it and run using chrome

Or you could check page on github pages:

`file:///C:/Users/dude/Desktop/bootcamp/UI/signin.html`

**Any background context you want to provide?**

None for now

**What are the relevant pivotal tracker stories?**

#163342847

### CODE REVIEW

**Comment by captainPrime** [comment]
**What does the PR do?**

Shows the signin page for the CRUD application

**Description of Task to be completed?**

Here i have the page working:

`file:///C:/Users/dude/Desktop/bootcamp/UI/signin.html`

**How should it be manually tested?**

After cloning the repo, cd into it and run using chrome

Or you could check page on github pages:

`file:///C:/Users/dude/Desktop/bootcamp/UI/signin.html`

**Any background context you want to provide?**

None for now

**What are the relevant pivotal tracker stories?**

#163342847

**Comment by ascii-dev** [comment]
Hi, kindly follow the commit message convention. Also, you don't need this URL `file:///C:/Users/dude/Desktop/bootcamp/UI/signin.html` because the path would definitely be different on my end. Lastly, you should include a link to your pivotal tracker story. Thanks

File: UI/signin.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                </div>
+                
+                <BR></BR>
+			</div>
+        </div>
+		<footer class="politico-footer">
+				<p>politico. &copy; 2019</p>
+			</footer>
+	</body>
+</html>
```
Comment (ascii-dev) [inline UI/signin.html:72]: A new line here, please

Comment (ascii-dev) [reply UI/signin.html:72]: Also, kindly format your html so it looks nice

**Pull request closed without merging**